subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/m5/ToolingApiIdeaModelCrossVersionSpec.groovy

```diff
@@ -317,8 +325,20 @@ project(':impl') {
         } else {
             libs.size() == 1
         }
-        libs.each {
-            it.dependencyModule == project.modules.find { it.name == 'api' }
+
+        if( currentVersion < GradleVersion.version('4.0')) {
```

Comment:
Let's use a consistent formatting pattern for `if` statements. We use `if (...)`.